angular-spa-ui/js/search-app/tags-module/tags-ctrl.js

```diff
@@ -1 +1,15 @@
 "use strict";
+
+module.exports = function($scope, $http) {
```

Comment:
As I remember, we decided to move _GET request to service. You can read this sample: http://stackoverflow.com/questions/17646034/what-is-the-best-practice-for-making-an-ajax-call-in-angular-js